storage/sqlite/store.go

```diff
@@ -0,0 +1,204 @@
+/*
+ * Flow Emulator
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sqlite
+
+import (
+	"context"
+	"database/sql"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	_ "github.com/glebarez/go-sqlite"
+	"github.com/onflow/flow-emulator/storage"
+)
+
+var _ storage.SnapshotProvider = &Store{}
+
+// Store implements the Store interface
+type Store struct {
+	storage.DefaultStore
+	db  *sql.DB
+	url string
+	mu  sync.RWMutex
+}
+
+func (s *Store) ListSnapshots() (snapshots []string, err error) {
```

Comment:
nit:
```suggestion
func (s *Store) Snapshots() (snapshots []string, err error) {
```
https://go.dev/doc/effective_go#Getters